packages/components/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "2.9.0",
+  "version": "2.9.1",
```

Comment:
not updated in lock file

Reply:
we never update the lockfile anyway, it's super annoying we have it anyway. you have to install from root package